pkg/broker/filter/filter_handler.go

```diff
@@ -232,7 +232,7 @@ func (r *Handler) sendEvent(ctx context.Context, tctx cloudevents.HTTPTransportC
 	if filterResult == failFilter {
 		r.logger.Debug("Event did not pass filter", zap.Any("triggerRef", trigger))
 		// Record the event count.
-		r.reporter.ReportEventCount(reportArgs, errors.New("event did not pass filter"))
+		r.reporter.ReportEventCount(reportArgs, http.StatusForbidden)
```

Comment:
maybe this is a 417 - (Expectation Failed)

Reply:
Done, thanks!